html_sanitizer.rst

```diff
@@ -314,6 +314,8 @@ attributes from the `W3C Standard Proposal`_ are allowed.
                             img: 'src'
                             # allow the <h1> element with all safe attributes
                             h1: '*'
+                            # allow the <div> element with no attributes
+                            div: []
```

Comment:
Can you also update xml, php and php-standelone code blocks to have this example ?

Thanks

Reply:
I would love to help, but I have no idea how to test XML config files.  I get nothing but DTD errors.